examples/taco/nodejs/src/index.ts

```diff
@@ -72,6 +73,17 @@ const encryptToBytes = async (messageString: string) => {
     encryptorSigner,
   );
 
+  // Calling it after the encryption because we need material from messageKit
```

Comment:
Since it is an example perhaps add something about this check not really being needed, so that users don't assume it needs to be added to their code.

```suggestion
  // Note: Not actually needed but used by CI for checking contract compatibility.
  // Calling it after the encryption because we need material from messageKit.
```